plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/wrapper/LibvirtPlugNicCommandWrapper.java

```diff
@@ -64,6 +64,12 @@ public Answer execute(final PlugNicCommand command, final LibvirtComputingResour
             final InterfaceDef interfaceDef = vifDriver.plug(nic, "Other PV", "", null);
             vm.attachDevice(interfaceDef.toString());
 
+            // apply default network rules on new nic
+            if (vmName.startsWith("i-") && nic.isSecurityGroupEnabled()) {
```

Comment:
Do we want to check this way if it's a User Instance? Doesn't have the VM itself have that property?